typescript/src/getBigUint64.ts

```diff
@@ -0,0 +1,18 @@
+// DataView.getBigUint64 was added to relatively recent versions of Safari. It's pretty easy to
```

Comment:
Maybe a polyfills file is a better home? Or compat?

Reply:
Also when working on the bag2proto it felt more natural for all the top level files in `src` to live in a _common_ folder since they otherwise add noise to browsing the project